### PR TITLE
adding merchant district encounters

### PR DESCRIPTION
**Pull request by ThomasLipp**

### CODE REVIEW

File: eldritch/encounters.py [hunk 1]
```diff
@@ -262,6 +262,111 @@ def Square7(char):
...
+  loss = events.Loss(char, {"sanity": 2})
+  # TODO: add prerequisite of ally being in deck
+  ally = events.DrawSpecific(char, "allies", "Brave Guy")
+  listen = events.Sequence([loss, ally], char)
+  return events.BinaryChoice(char, "Listen to the tale?", "Yes", "No", listen, events.Nothing())
+def Unnamable2(char):
+  check = events.Check(char, "lore", -1)
+  spell = events.Draw(char, "spells", 1)
+  clues = events.Gain(char, {"clues": 2})
+  delayed = events.LoseTurn(char)
```
Comment (fingoldfish) [inline eldritch/encounters.py:316]: This one should be Delayed, not LoseTurn.

File: eldritch/encounters.py [hunk 2]
```diff
@@ -262,6 +262,111 @@ def Square7(char):
...
+  return events.PassFail(char, check, move, loss)
+def Unnamable5(char):
+  check = events.Check(char, "luck", -1)
+  unique = events.Draw(char, "unique", 1)
+  loss = events.Loss(char, {"sanity": 1, "stamina": 2})
+  return events.PassFail(char, check, unique, loss)
+def Unnamable6(char):
+  check = events.Check(char, "speed", -1)
+  lost = events.Nothing() #TODO: I'm stuck inside a clock and I'm somewhere around Mars?
+  return events.PassFail(char, check, events.Nothing, lost)
```
Comment (fingoldfish) [inline eldritch/encounters.py:335]: This one should be events.Nothing() instead of events.Nothing

File: eldritch/test_encounters.py [hunk 3]
```diff
@@ -849,5 +851,374 @@ def testTrain3Split(self):
...
+    # we know these aren't actually spells
+    self.state.spells.extend([items.Cross(), items.HolyWater()])
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "Yes")
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=2)):
+      self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 0)
+    self.assertEqual(len(self.state.spells), 2)
+    self.assertEqual(self.char.clues, 2)
+    self.assertEqual(self.char.lose_turn_until, self.state.turn_number + 2)
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:1066]: delayed_until

**Review by fingoldfish — APPROVED** [review]

File: eldritch/encounters.py (hunk 1 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: not sure why I didn't do that as delayed, fixed.

File: eldritch/test_encounters.py (hunk 3 above)
Comment (ThomasLipp) [reply eldritch/test_encounters.py]: fixed

File: eldritch/encounters.py (hunk 2 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: oops, that's what's happened when you don't have a unit test (and an uncompiled language)

**Pull request merged**